src/components/markdown/PGS.ts

```diff
@@ -26,9 +26,9 @@ export default {
     '6.2': 31,
     '6.3': 31,
     '7': 33,
-    '7.1': 33,
-    '7.2': 33,
-    '7.3': 34,
+    '7.1': 31,
```

Comment:
no way this is correct? the numbers are no longer in increasing order ...

Reply:
it's no longer in increasing order, but it's [correct](https://darrenyao.com/usacobook/cpp.pdf#page=31), I believe the others have to be updated as well

Reply:
i'll look into it more i think a shift in pages occurred

Reply:
er ok it seems like all the pages are shifted by a non-constant amount, do you want me to renumber the entire file

Reply:
Yes. I believe that I copied the table of contents and used a Python script to extract the section and page numbers, but I'm not sure where it is anymore.